src/board/system76/darp8/board.mk

```diff
@@ -7,6 +7,9 @@ CFLAGS+=-DEC_ESPI=1
 # Apply PMC hack for S0ix
 CFLAGS+=-DPMC_S0IX_HACK=1
 
+# Use S0ix
+CFLAGS+=-DUSE_S0IX=1
+
```

Comment:
We're not using S0ix on these devices anymore, please remove

Reply:
Done.